package.json

```diff
@@ -1,121 +1,121 @@
 {
-  "name": "react-accessible-accordion",
```

Comment:
Prettier has changed the whitespace here, disable whitespace to see more meaningful changes.